package.json

```diff
@@ -9,20 +9,19 @@
   },
   "scripts": {
     "build": "ember build",
-    "coverage": "COVERAGE=true ember test -r 'dot'",
```

Comment:
was causing tests to hang like we saw for core. will try to investigate if it is an acceptance test issue again